--- mozplugger-2.1.6/Makefile.in.old	2015-02-11 13:12:28.790025563 +0100
+++ mozplugger-2.1.6/Makefile.in	2015-02-11 13:13:44.313362647 +0100
@@ -267,17 +267,17 @@
 	make localinstall BROWSERDIR=.opera
 
 install:
-	-install -d @bindir@
-	install mozplugger-helper @bindir@
-	install mozplugger-controller @bindir@
-	install mozplugger-linker @bindir@
-	install mozplugger-update @bindir@
-	-for a in ${PLUGINDIRS}; do install -d $$a ; done
-	for a in ${PLUGINDIRS}; do install mozplugger.so $$a ; done
-	-install -d @sysconfdir@
-	install $(srcdir)/mozpluggerrc @sysconfdir@
-	-install -d @mandir@/man7
-	install $(srcdir)/mozplugger.7 @mandir@/man7/
+	-install -d $(DESTDIR)@bindir@
+	install mozplugger-helper $(DESTDIR)@bindir@
+	install mozplugger-controller $(DESTDIR)@bindir@
+	install mozplugger-linker $(DESTDIR)@bindir@
+	install mozplugger-update $(DESTDIR)@bindir@
+	-for a in ${PLUGINDIRS}; do install -d $(DESTDIR)$$a ; done
+	for a in ${PLUGINDIRS}; do install mozplugger.so $(DESTDIR)$$a ; done
+	-install -d $(DESTDIR)@sysconfdir@
+	install $(srcdir)/mozpluggerrc $(DESTDIR)@sysconfdir@
+	-install -d $(DESTDIR)@mandir@/man7
+	install $(srcdir)/mozplugger.7 $(DESTDIR)@mandir@/man7/
 
 uninstall:
 	-rm -f @sysconfdir@/mozpluggerrc
